src/modules/workflowsettings/Elsa.WorkflowSettings.Persistence.YesSql/Documents/WorkflowSettingsDocument.cs

```diff
@@ -0,0 +1,12 @@
+using Elsa.Persistence.YesSql.Documents;
+
+namespace Elsa.WorkflowSettings.Persistence.YesSql.Documents
+{
+    public class WorkflowSettingsDocument : YesSqlDocument
```

Comment:
I think this should be singular (`WorkflowSettingDocument`) since it represents a single setting, right? Same goes for things like `WorkflowSettingsIndex`.